pkg/skaffold/build/cache/hash.go

```diff
@@ -37,6 +37,10 @@ import (
 	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/util"
 )
 
+const (
+	storePrefix = "dependency_hash"
```

Comment:
why is the `storePrefix` needed?

Reply:
The reason, i added storePrefix is to gaurantee there is no collision of keys if the `util.SyncStore` was used by different modules.

Reply:
I think between different instances of `*SyncStore` getting same keys wouldn't matter right since they'd be using different instances of both the `singleflight.Group` and `sync.Map`?

Reply:
you are right! Thanks. i will drop it :)